examples/ImageStreamer/README.md

```diff
@@ -0,0 +1,217 @@
+# Ansible for HPE OneView and Image Streamer
+
+This repository contains a collection of sample ansible playbooks for using HPE Image Streamer to deploy, update and revert Golden Images to OS boot/run volumes for HPE Synergy compute module servers.
```

Comment:
Maybe you could check this sentence "to OS boot/run volumes for HPE Synergy compute module servers", to improve a little bit.